src/rpc/server.cpp

```diff
@@ -456,6 +456,11 @@ static inline JSONRPCRequest transformNamedArguments(const JSONRPCRequest& in, c
             hole += 1;
         }
     }
+    auto wallet = argsIn.find("wallet");
+    if (wallet != argsIn.end() && wallet->second->isStr()) {
```

Comment:
I would say `wallet` argument *must* be a string otherwise `RPC_INVALID_PARAMETER` should be raised.